addons/mass_mailing_event/models/event_event.py

```diff
@@ -16,7 +16,7 @@ def action_mass_mailing_attendees(self):
             'target': 'current',
             'context': {
                 'default_mailing_model_id': self.env.ref('event.model_event_registration').id,
-                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', '!=', 'cancel')])
+                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', '!=', 'cancel'), ('state', '!=', 'draft')])
```

Comment:
Why not state, not in, ... ?

Reply:
Thought it'd look better in the domain selector but it's fine actually